lib/tasks/submissions.rake

```diff
@@ -5,3 +5,32 @@ task :exchange_assigned_to_real_user => :environment do
     submission.exchange_assigned_to_real_user!
   end
 end
+    
+task correction_seq_id: :environment do
+  ActiveRecord::Base.connection.tables.each do |t|
+    ActiveRecord::Base.connection.reset_pk_sequence!(t)
+  end
+end
+
+task :rebase_user_submission => :environment do
```

Comment:
If possible, I'd like to run this task together on staging once this PR is merged and ready.